generate.md

```diff
@@ -27,46 +27,54 @@ Run:
 
 *Note, if pip doesn't work depending on your system is configured you man need to replace `pip` with `pipx`, `pip3`, or `python -m pip`.*
 
-*Also note: The `-e` is optional. It allows you to adjust the code in `nimble-builder` or `nimble-orchestration` without having to run the installation again.*
-
 
 ## Running the code
 
-As the code is very much a work in progress we do not have detailed documentation of how to run it for custom configurations.
+### Lunching the configuration server
 
-For now the best way to run the code is with the two scripts in this repository.
+Starting in the root directory of this project. First you will need to generate the configuration options. To do this run:
 
-### Generate static
+    gen_nimble_conf_options
 
-The script `generate-static.py` is used to create a number of example components for a Nimble rack. The script is not trying to support any specific configuration, but is instead being developed to provide a static library interface for which a large number of Nimble components can be downloaded.
+Next run:
 
-To run this script, run the following command:
+    cadorchestrator serve
 
-    python generate_static.py
+The server should now be available at http://127.0.0.1:8000/
 
-This should create the `build` directory. Inside this the `printed_components` directory should contain a number of `stl` files that can be 3D printed.
+### Generating a specific configuration from command line
 
-The script also creates a simple web-page with an index of all of the files. The final link to the "Partially complete automatically generated documentation" will be broken unless you also run `generate.py` (see below).
+The script `cadorchestrator generate` can be used generating a complete set of information for a nimble configuration from the command line.
 
-It also creates a number of files that are used by the orchestration script.
+Currently the configuration is just a list of the networking components which are hard coded. At a future date it should be possible to pass a configuration and other parameters to this script.
 
-If you don't want to run this locally you can see a [hosted version of the output](https://wakoma.github.io/nimble/).
+To generate run the following:
 
+    cadorchestrator generate '["NUC10i5FNH", "Raspberry_Pi_4B", "Raspberry_Pi_4B"]'
 
-### Generate configuration
+This should create the `build` directory. Inside this the `printed_components` directory should contain a number of `step` files that can be 3D printed (you may need to convert them to `stl` files first).
```

Comment:
Why are we not exporting STL files for the `printed_components` directory?

Reply:
This sentence also seems to be a duplicate of line 72 with the exception that 72 mentions STL files.

Reply:
72 is describing what happens in `generate_static.py` which is why things are repeated. Eventually we can get rid of `generate_static.py` but I think it still serves a purpose.

I agree this is a mistake as we do create both STLs and STEPs. I'll fix the text.

As for why don't we simply create STLs. I think the main issue is that CadQuery can't import STLs (or if it can, I don't know how to do it!!) so we can't assemble them. This is why the main generation needs to make both STEPs and STLs.

Reply:
I believe there is a plugin that allows STLs to be imported, but STL->STEP conversion can get pretty messy if it is a complex shape with non-planar faces.

Reply:
I've fixed the offending line. I've also added issue #83 to discuss details